src/main/resources/amailp/intellij/robot/lexer/Robot.flex

```diff
@@ -77,6 +78,7 @@ EnvironmentVariable = "%{" ~"}"
         {Word} | "$" | "@" | "&" | "%"  { yybegin(LINE); return RobotTokenTypes.Word; }
         {Space}                         { yybegin(LINE); return RobotTokenTypes.Space; }
         {Separator}                     { yybegin(LINE); return RobotTokenTypes.Separator; }
+        {LibraryAliasSeparator}         { yybegin(LINE); return RobotTokenTypes.LibraryAliasSeparator; }
```

Comment:
[minor] Personally I do not like to much the `LibraryAliasSeparator` name, since `Separator` has a well defined meaning in the plugin at the moment (what separates Cells). I would have probably called id `WithName` since, at lexing time, it is not more than a very specific token.

But feel free to keep it if you have a strong opinion against :-)

Reply:
I also doubted this question. There’s nothing to argue about :)